service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ReturnApplicationAboutToSubmitTest.java

```diff
@@ -52,6 +52,36 @@ void shouldMigrateSubmittedDocumentToReturnedDocumentBundle() {
         assertThat(extractedCaseData.getReturnApplication()).isEqualTo(expectedReturnApplication);
     }
 
+    @Test
+    void shouldMigrateSubmittedDocumentToReturnedDocumentBundleIfMainApplicationIsRemovedBySupeAdmin() {
```

Comment:
Typo in test name at the end, missing R in SuperAdmin

Reply:
Fixed!